src/framework/components/gsplat/component.js

```diff
@@ -38,11 +39,37 @@ import { GSplatPlacement } from '../../../scene/gsplat-unified/gsplat-placement.
  * console.log(entity.gsplat.customAabb);
  * ```
  *
+ * ## Unified Rendering
+ *
+ * The {@link GSplatComponent#unified} property enables unified rendering mode, which provides
+ * advanced features for Gaussian Splats:
+ *
+ * - **Global Sorting**: Multiple splat components are sorted together in a single unified sort,
+ *   eliminating visibility artifacts and popping effects when splat components overlap.
+ * - **LOD Streaming**: Dynamically loads and renders appropriate levels of detail based on camera
+ *   distance, enabling efficient rendering of massive splat scenes.
+ *
+ * ```javascript
+ * // Enable unified rendering for advanced features
+ * entity.gsplat.unified = true;
+ * ```
+ *
+ * Note: The `unified` property can only be changed when the component is disabled.
```

Comment:
I know I mentioned this before, but this still seems super-weird to me. If you change it on an enabled component, we should just do whatever's necessary internally to make it work.